easybuild/easyconfigs/a/ABySS/ABySS-1.9.0-foss-2016a.eb

```diff
@@ -0,0 +1,35 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Maxime Schmitt <maxime.schmitt@telecom-bretagne.eu>, Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'ABySS'
+version = '1.9.0'
+
+homepage = 'http://www.bcgsc.ca/platform/bioinfo/software/abyss'
+description = """Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'usempi': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['https://github.com/bcgsc/abyss/releases/download/%(version)s/']
+
+dependencies = [
+	('Boost', '1.60.0'),
+	('sparsehash', '2.0.2')
```

Comment:
please avoid using tabs for indentation, use spaces instead (one indent level == 4 spaces)